Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "openzeppelin-polkadot-wrappers"
+authors = [ "OpenZeppelin" ]
+description = "Wrappers with Sensible Defaults for constructing Polkadot Parachain Runtimes"
+edition = "2021"
+license = "GPL-3.0-only"
+repository = "https://github.com/OpenZeppelin/polkadot-runtime-wrappers"
+readme = "README.md"
+version = "0.0.1"
```

Don't you think 0.1.0 is more appropiate? Still signals that there will be breaking changes, but gives is the opportunity to issue patch releases(e.g. 0.1.1, 0.1.2, etc.)
```suggestion
version = "0.1.0"
```